components/vector-kubearchive-log-collector/base/vector-pre.yaml

```diff
@@ -98,6 +100,12 @@ subjects:
 - kind: ServiceAccount
   name: vector-kubearchive-log-collector-loki-canary
   namespace: product-kubearchive-logging
+- kind: ServiceAccount
+  name: minio-sa
+  namespace: product-kubearchive-logging
+- kind: ServiceAccount
+  name: vector-kubearchive-log-collector-minio
+  namespace: product-kubearchive-logging
```

Comment:
minio should only be used in development, so this should be set only under the `development` directory, right?